f8a_worker/defaults.py

```diff
@@ -3,6 +3,25 @@
 from urllib.parse import quote
 
 
+def _make_postgres_string(password=''):
+    """
+    Method creates postgres connection string. It's parametrized, so it's possible to create either
+    quoted or unquoted version of connection string.
+    Note that it's outside of class since there is no simple way how to call it inside the class
+    without class initialization.
```

Comment:
What about marking it as `@staticmethod` and calling it `F8AConfiguration.make_postgres_string()` ?

Reply:
Chicken or the egg problem. `F8AConfiguration` is not defined in a body of `F8AConfiguration`.

Reply:
It would be in the same namespace so something like this should do the trick:

```python
class A(object):

    def _make_foo():
        return "bar"

    FOO = _make_foo()


print(A.FOO)
```

Also note that it needs to be defined before use.

Reply:
OK, you seem to be right. [Calling a static method from another method works](https://stackoverflow.com/questions/35459360/how-to-call-static-methods-inside-the-same-class-in-python), but we need to call it during initialization of the class, which is a no-go.

Reply:
Nice, I like that!, I don't even have to expose FOO in my case :-)